style.css

```diff
@@ -1,37 +1,263 @@
-* {
-  max-width: 700px;
-  margin: auto;
+body {
+    font-family: Arial, sans-serif;
+    line-height: 1.6;
 }
 
-#title {
-  margin-left: 3%;
+.container {
+    width: 80%;
+    margin: auto;
+    overflow: hidden;
 }
 
-h1 {
-  color: red;
+.header {
+    display: flex;
+    align-items: center;
 }
 
-p,
-li {
-  font-family: "Cormorant Garamond";
+.profile-photo {
+    width: 100px;
+    height: 100px;
+    border-radius: 50%;
+    margin-right: 20px;
+}
+
+.grid {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 20px;
 }
 
 .head {
-  font-size: 20px;
+    font-weight: bold;
+    font-size: 30px;
+}
+
+h1,
+h2,
+h3 {
+    padding-bottom: 12px;
+}
+
+.job,
+.education {
+    margin-bottom: 5px;
+}
+
+.job h3 {
+    margin-bottom: -16px;
+    margin-top: 30px;
+
+}
+
+.job p {
+    line-height: 1.2;
+
+}
+
+hr {
+    margin: 20px 0;
 }
 
 a {
-  color: black;
-  text-decoration: none;
+    color: #333;
+    text-decoration: none;
 }
 
-@media only screen and (max-width: 430px) {
-  .left,
-  .right {
-    display: none;
-  }
-  .stuff {
-    width: 100%;
+a:hover {
+    color: #666;
+    text-decoration: underline;
+}
+
+ul {
+    padding: 0;
+    list-style-type: none;
+}
+
+.social-icons {
+    display: flex;
+    justify-content: flex-start;
+    margin-top: -36px;
+    margin-left: 127px;
+}
+
+.github-icon,
+.linkedin-icon {
+    position: relative;
+    text-align: left;
+}
+
+.github-icon {
+    left: -10px;
+    top: 0px;
+}
+
+.linkedin-icon {
+    left: -10px;
+    top: 0px;
     margin-left: 10px;
-  }
+}
+
+.icon {
+    width: 32px;
+    height: 32px;
+    transition: all 0.3s ease;
+}
+
+.github-icon:hover .icon,
+.linkedin-icon:hover .icon {
+    transform: scale(1.2);
+    opacity: 0.7;
+}
+
+.blog-link {
+    text-decoration: none;
+    color: #333;
+    letter-spacing: normal;
+    transition: all 0.3s ease;
+}
+
+.blog-link:hover {
+    letter-spacing: 1px;
+    text-decoration: underline;
+    text-decoration-thickness: 2px;
+}
+
+.education-link {
+    text-decoration: none;
+    color: #333;
+    letter-spacing: normal;
+    transition: letter-spacing 0.3s ease;
+}
+
+.education-link:hover {
+    letter-spacing: 1px;
+}
+
+.focus-areas {
+    padding: 6px;
+    border: 1px solid #f5f5f5;
+    background-color: #f5f5f5;
+    font-size: 14px;
+
+}
+.title-focus {
+    color: #333;
+    font-size: 14px;
+    font-weight: bold;
+    margin-bottom: 10px;
+}
+
+.blog-title {
+    font-size: 30px;
+    color: #333;
+}
+
+.tooltip {
+    display: none;
+    position: absolute;
+    background-color: rgba(0, 0, 0, 0.8);
+    color: white;
+    padding: 5px;
+    border-radius: 3px;
+    font-size: 14px;
+    z-index: 100;
+}
+
+li {
+    position: relative;
+    cursor: pointer;
+}
+
+
+
+
+.focus-areas span {
+    display: inline-block;
+    transition: transform 0.3s ease;
+}
+
+
+
+/* Стиль кнопок тегов */
```

Comment:
Can you please translate to English?

Reply:
Here, I specified the style for the buttons, which I later removed from the code